docs/source/tutorials/stable_diffusion.mdx

```diff
@@ -352,4 +352,46 @@ To avoid Neuron device out of memory, it's suggested to finish all base inferenc
 </Tip>
 
 
+## Latent Consistency Models
+
+Latent Consistency Models (LCMs) were proposed in [Latent Consistency Models: Synthesizing High-Resolution Images with Few-Step Inference by Simian Luo, Yiqin Tan, Longbo Huang, Jian Li, and Hang Zhao](https://huggingface.co/papers/2310.04378). LCMs enable inference with fewer steps on any pre-trained LDMs, including Stable Diffusion and SDXL.
+
+Here is an example to compile a LCM of Stable Diffusion XL: [latent-consistency/lcm-sdxl](https://huggingface.co/latent-consistency/lcm-sdxl) and run inference on AWS Inferentia 2 :
+
+### Compile LCM
+
+***Export via APIs***
+
+```python
+from optimum.neuron import NeuronStableDiffusionXLPipeline
+
+model_id = "stabilityai/stable-diffusion-xl-base-1.0"
+unet_id = "latent-consistency/lcm-sdxl"
+num_images_per_prompt = 1
+input_shapes = {"batch_size": 1, "height": 1024, "width": 1024, "num_images_per_prompt": num_images_per_prompt}
```

Comment:
I would just make sure it fits in a single line in the documentation, or reformat that in multiple lines otherwise.

Reply:
I think it's ok, users can scoll right with the bar in our doc. I would prefer to keep the snippet looking short as this doc is already long.